ci/release/_install.sh

```diff
@@ -252,19 +309,39 @@ install_standalone_d2() {
   "$sh_c" sh -c "'cd \"$INSTALL_DIR/d2-$VERSION\" && make install PREFIX=\"$PREFIX\"'"
 }
 
-install_tala() {
-  REPO="${REPO_TALA:-terrastruct/TALA}"
+install_d2_brew() {
+  header "installing d2 with homebrew"
+  sh_c brew tap terrastruct/d2
+  sh_c brew install d2
+  sh_c brew test d2
```

Comment:
my brew gave a warning? r we supposed to run this post-install for everyone?

![Screen Shot 2022-11-20 at 7 05 51 PM](https://user-images.githubusercontent.com/3120367/202954677-56c99cfa-8f44-46c7-98d2-1afa60d5a1f7.png)

Reply:
Oh I must have dev mode on by default. Ok, in that case I'll just remove `brew test`.